<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>profiling_gui::MainWindow</class>
 <widget class="QMainWindow" name="profiling_gui::MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>887</width>
    <height>595</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Profiling visualizer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="profiling_gui::HorizontalView" name="mainView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>887</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QDockWidget" name="coroutinesDock">
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QTreeView" name="coroutineView"/>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>Open prifiling data</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>profiling_gui::HorizontalView</class>
   <extends>QGraphicsView</extends>
   <header>profiling_gui/horizontalview.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
